examples/nodejs/token-vending-machine/lambda/authorizer/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
npm sometimes does very weird/unexpected things when you have a package.json file in a subdirectory of another dir that also has a package.json.

Roughly: https://flaviocopes.com/npm-install-packages-parent-folder/

So I would recommend creating a new subdir `lambda/token-vending-machine` for the previous lambda code, and then you can have that as a sibling dir for your new `authorizer` dir.